### PR TITLE
Fix/team screen

### PR DESCRIPTION
**Pull request by subbotnik16**

### CODE REVIEW

File: src/screens/Team/components/ContactsList/hooks/useGetContacts.ts [hunk 1]
```diff
@@ -21,23 +17,15 @@ export const useGetContacts = () => {
   }, [contactsIds.length, dispatch]);
 
   const sections = [
+    {
+      title: 'iceFriends',
```
Comment (victor-oancea-leftclick-io) [inline src/screens/Team/components/ContactsList/hooks/useGetContacts.ts:21]: I18n

File: src/screens/Team/components/ContactsList/index.tsx [hunk 2]
```diff
@@ -38,9 +40,16 @@ export const ContactsList = ({}: ContactsListProps) => {
 
   const invite = useCallback(
     (id: string) => {
+      hapticFeedback();
+      const contact = contactsByIds[id];
```
Comment (pisarevkir) [inline src/screens/Team/components/ContactsList/index.tsx:44]: that should be in sagas (everything below `hapticFeedback()`)

File: src/screens/Team/components/ContactsList/hooks/useGetContacts.ts (hunk 1 above)
Comment (subbotnik16) [reply src/screens/Team/components/ContactsList/hooks/useGetContacts.ts:21]: no, we shouldn't it's just a string which i compare in the component to show a specific section header

File: src/store/modules/Team/sagas/inviteContactSaga.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import {put, select} from 'redux-saga/effects';
+
+const actionCreator = TeamActions.INVITE_CONTACT.START.create;
+
+export function* inviteContactSaga(action: ReturnType<typeof actionCreator>) {
+  try {
+    const {id} = action.payload;
+    const contactsByIds: ContactById = yield select(getContactsByIdsSelector);
+    const contact = contactsByIds[id];
+    const text = `${t('team.contacts_list.hi')} ${contact.firstName}, ${t(
```
Comment (victor-oancea-leftclick-io) [inline src/store/modules/Team/sagas/inviteContactSaga.ts:17]: use only 1 text

**Review by ice-ares — CHANGES_REQUESTED** [review]

**Review by victor-oancea-leftclick-io — APPROVED** [review]

**Review by pisarevkir — DISMISSED** [review]

**Review by victor-oancea-leftclick-io — APPROVED** [review]

**Review by OlehPanforov — APPROVED** [review]

**Pull request merged**